website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -122,30 +122,16 @@ base branch.
 To delete a published tag to re-tag, follow these steps:
 
 ```
-    git tag -d vx.y.z
-    git push -d origin vx.y.z
+    git tag -d vx.y.z
+    git push -d origin vx.y.z
 ```
 
 ## Deploying ORCA buckets, RDS cluster and Cumulus ORCA modules via bamboo
 
-For testing purposes, you should use your feature branch in cumulus-orca github repo and [`prototype-latest`](https://ci.earthdata.nasa.gov/browse/ORCA-PL) bamboo plan so that it does not affect the ORCA github `develop` branch. The `prototype-latest` bamboo plan is linked to `feature/ORCA-test-bamboo` github branch as the default branch under cumulus-orca repo. In addition, if changes are made to the bamboo spec file `bamboo.yaml` in this default branch, you have to manually import the bamboo spec by choosing `orca test branch` as the linked repo.
-
-:::warning
-You should reset `feature/ORCA-test-bamboo` before using it.
-1. Rename `feature/ORCA-test-bamboo` to `feature/ORCA-test-bamboo-old`
-1. Create a new branch based off of your branch named `feature/ORCA-test-bamboo`
-1. In the new branch's `bamboo.yaml`:
-   Delete all but one of the plans and the `ORCA-ODP` plan.
-   Change plan's `name` to `prototype-latest`.
-   In each `repositories` element, change `orca-develop` to `orca test branch`.
-   In each `plan` element, change `OI`/`ODP` in `key` values to `PL`
-:::
+For testing purposes, you should use your feature branch in cumulus-orca github repo and [`Test ORCA Deploy Plan`](https://ci.earthdata.nasa.gov/browse/ORCA-TODP) bamboo plan so that it does not affect the ORCA github `develop` branch. The `Test ORCA Deploy Plan` bamboo plan is linked to `feature/ORCA-test-bamboo` github branch as the default branch under cumulus-orca repo.

```

Comment:
Is there a way to run the Cleanup plan?

Reply:
Unfortunately, not in bamboo since it requires running ssh and ssm commands whcih is not feasible as per my knowledge.

Reply:
So the Cleanup plan doesn't work anywhere?

Reply:
Could've sworn I've used it to remove integration test components via Bamboo.

Reply:
Oh I thought it's for cleaning up the database. 
Yes, `ORCA Cleanup Plan` takes care of the cleanup step. This is discussed at end of the doc already.

Reply:
These steps are for deploying. For cleanup, the steps are already added in the document at the bottom section.

Reply:
Can you run the cleanup plan with an arbitrary branch's version of the plan?

Reply:
Yes, you can.

Reply:
How? These docs make no mention of this. They only mention the `ORCA Cleanup Plan` which is based off of `develop`, not an arbitrary branch.

Reply:
Will add to the doc.